index.js

```diff
@@ -18,4 +20,339 @@ const openMenu = document.querySelector('.button');
+                 <li class="tags">ruby</li>
+             </ul>
+             </div>
+             <div class="hov2">
+             <button class="project-button" id="probutton3">See Project</button>
+             </div>
+             </div>
+             <div class="card" id="procard3">
+                 <div class="hov1">
+                 <h2 class="cardhead">Profesional Art Printing Data More</h2>
+                 <h2 class="cardhead1"  id="prohead3">Profesional Art Printing Data</h2>
+                 <p class="cardpara" id="propara3">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                 <ul class="card-list" id="pro-list3">
+                     <li class="tags">html</li>
+                     <li class="tags">bootstrap</li>
+                     <li class="tags">ruby</li>
+                 </ul>
+                 </div>
+                 <div class="hov2">
+                 <button class="project-button" id="probutton4">See Project</button>
+                 </div>
+                </div>
+                 <div class="card"  id="procard4">
+                     <div class="hov1">
+                     <h2 class="cardhead">Profesional Art Printing Data</h2>
+                     <h2 class="cardhead1" id="prohead4">Data Dashboard Healthcare</h2>
+                     <p class="cardpara" id="propara4">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                     <ul class="card-list" id="pro-list4">
+                         <li class="tags">html</li>
+                         <li class="tags">bootstrap</li>
+                         <li class="tags">ruby</li>
+                     </ul>
+                     </div>
+                     <div class="hov2">
+                     <button class="project-button" id="probutton5">See Project</button>
+                     </div>
+                     </div>
+                     <div class="card"  id="procard5">
+                         <div class="hov1">
+                         <h2 class="cardhead">Profesional Art Printing Data</h2>
+                         <h2 class="cardhead1" id="prohead5">Website Protfolio </h2>
+                         <p class="cardpara" id="propara5">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                         <ul class="card-list" id="pro-list5">
+                             <li class="tags">html</li>
+                             <li class="tags">bootstrap</li>
+                             <li class="tags">ruby</li>
+                         </ul>
+                         </div>
+                         <div class="hov2">
+                         <button class="project-button" id="probutton6">See Project</button>
+                         </div>
+                         </div>
+ </div>
+</div>
+</div>`);
```

Comment:
- [ ] Well done removing the code from HTML and adding it with JS, but completing the works section is more complicated than copying and pasting the code from HTML, please 🙏🏽  implement an array of objects to store JavaScript objects containing the information for each project. Each object should contain at least the following information about the project: name, description, featured image, technologies, and button. After that, the projects section is created dynamically with the information from the projects array. Thanks to that, adding projects in the future will be easy.